packages/sign-client/canary/canary.spec.ts

```diff
@@ -0,0 +1,53 @@
+import "mocha";
+import {
+  initTwoClients,
+  testConnectMethod,
+  deleteClients,
+} from "./../test/shared";
+import CloudWatch from 'aws-sdk/clients/cloudwatch';
+
+const environment = process.env.ENVIRONMENT || 'dev';
+
+describe("Canary", function() {
+  describe("HappyPath", function() {
+    // TODO: implement a test that depicts
+    // the happy case better
+    it("connects", async function() {
+      const clients = await initTwoClients();
+      await testConnectMethod(clients);
```

Comment:
This feels like it's leaving the door open on false positive passes since we're not doing any concrete assertions here, i.e. relying on something to throw in these setup functions.

It's not a silver bullet but would be nice to additionally validate the end result with assertions:

```js
expect(clients.A instanceof SignClient).toBe(true);
expect(clients.B instanceof SignClient).toBe(true)
```